charts/datadog/templates/cluster-agent-rbac.yaml

```diff
@@ -114,6 +114,7 @@ rules:
   resources:
   - deployments
   - replicasets
+  - daemonsets
```

Comment:
It is not needed to add `daemonsets` in this list because the cluster-agent should try to access `daemonsets` only if the `.Values.clusterAgent.admissionController.enabled` is `true`, and so your change [line#153](https://github.com/DataDog/helm-charts/pull/129/files#diff-0274b23790867ea01728a74abbce7f26a5686634bcb7c59fe04ecf92a81589b0R153) is enough.